packages/core/test/unit/createApp.test.ts

```diff
@@ -144,5 +144,16 @@ describe('createApp', () => {
 			await app.start();
 			should(app.server).have.property('listen');
 		});
+		it('Should successfully configure a port with middleware', async () => {
```

Comment:
nit: a better test description could be: 
'Should successfully configure a http express app with a custom port'